docs/csharp/roslyn-sdk/source-generators-overview.md

```diff
@@ -72,6 +72,8 @@ In this guide, you'll explore the creation of a source generator using the <xref
 
     > [!TIP]
     > The source generator project needs to target the `netstandard2.0` TFM, otherwise it will not work.
+    > The source generator can be hosted under .NET Framework environment (in Visual Studio IDE or MSBuild CLI), or .NET 7.0+ environment (`dotnet` CLI, VS Code, or VS out-of-process).
```

Comment:
@jaredpar in case you got time to review the technicalities here

Reply:
`NET 6.0` is still a valid target right now. The compiler can also be hosted by Mono still which in turn means the same for analyzers.